### DISCUSSION TITLE
Display Field option based on other field

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#16564] [Archive]
I would like to change the color of the field CA_01 based of the other field (in the same table) CA_01N1

Is that possible?
![Capture d’écran 2022-11-21 à 17 20 06](https://user-images.githubusercontent.com/4217180/203105819-de5a73ea-fafa-4746-9a53-854df5d434cf.png)

**Comment by br41nslug** [comment]
That is currently not possible.

**Reply by nervusdm** [reply]
Sadly, I see.

**Reply by azatTemirbek** [reply]
Seems like we have to inject form values on apply rules function
